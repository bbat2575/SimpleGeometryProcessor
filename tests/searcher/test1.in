-17, 10
-20, 11
32, -49
48, -36
7, 45
-9, -0
31, -38
42, -4
22, -40
45, -7
-6, 37
-15, 37
-42, -28
18, -33
-12, -47
-4, 47
19, 26
5, 16
48, -44
8, 32
