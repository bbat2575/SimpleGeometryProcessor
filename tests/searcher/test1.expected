read 20 points
31.00, -38.00
22.00, -40.00
18.00, -33.00
This is a triangle
